Functions/Admin/Backup-RsEncryptionKey.ps1

```diff
@@ -4,71 +4,134 @@
 
-    .PARAMETER SqlServerInstance (optional)
-        Specify the name of the SQL Server Reporting Services Instance.
-
-    .PARAMETER SqlServerVersion (optional)
-        Specify the version of the SQL Server Reporting Services Instance. 13 for SQL Server 2016, 12 for SQL Server 2014, 11 for SQL Server 2012
-
-    .PARAMETER Password
-        Specify the password to be used for backing up the encryption key. This password will be required when restoring the encryption key.
+	    .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ComputerName
+            The Report Server to target.
+            Use the "Connect-RsReportServer" function to set/update a default value.
         
-    .PARAMETER KeyPath
-        Specify the path to where the encryption key should be stored. 
+        .PARAMETER Credential
+            The credentials with which to connect to the Report Server.
+            Use the "Connect-RsReportServer" function to set/update a default value.
 
-    .EXAMPLE
-        Backup-RSEncryptionKey -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
-        Description
-        -----------
-        This command will back up the encryption key against default instance from SQL Server 2016 Reporting Services
-    
-    .EXAMPLE
-        Backup-RSEncryptionKey -SqlServerInstance 'SQL2012' -SqlServerVersion '11' -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
-        Description
-        -----------
-        This command will back up the encryption key against named instance (SQL2012) from SQL Server 2012 Reporting Services 
+	    .PARAMETER Password
+	        Specify the password to be used for backing up the encryption key. This password will be required when restoring the encryption key.
+	        
+	    .PARAMETER KeyPath
+	        Specify the path to where the encryption key should be stored. 
+
+	    .EXAMPLE
+	        Backup-RSEncryptionKey -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
+	        Description
+	        -----------
+	        This command will back up the encryption key against default instance from SQL Server 2016 Reporting Services
+	    
+	    .EXAMPLE
+	        Backup-RSEncryptionKey -ReportServerInstance 'SQL2012' -ReportServerVersion '11' -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
+	        Description
+	        -----------
+	        This command will back up the encryption key against named instance (SQL2012) from SQL Server 2012 Reporting Services
+	
+		.NOTES
+			Author:      ???
```

Comment:
Typically we rely in the source control for the changes to individual functions instead of keeping a log inside the function, the latest are not likely to be updated in incremental changes and add "clutter" to the code that difficults the readability